ui/src/App.js

```diff
@@ -107,15 +107,15 @@ class App extends Component {
     }
 
     serializeFilters(searchFilters) {
-        let filterStr = [];
+        let filterList = [];
         searchFilters.forEach((values, key) => {
             if (values.length > 0) {
                 for (let value of values) {
-                    filterStr.push(key + "=" + value);
+                    filterList.push(key + "=" + value);
                 }
             }
         });
-        return filterStr.join(',');
+        return filterList;
     }
 }
 
```

Comment:
Adding comment here so you can reply to it if you want.

I git checkout and I see this bug:
* Check a facet. Things work as expected.
* Uncheck same facet. Get console error and UI doesn't update.

Reply:
This is a separate issue which I have a followup PR staged for.